explorer/nodewatch/nodewatch/__init__.py

```diff
@@ -91,6 +91,17 @@ def symbol_summary():  # pylint: disable=unused-variable
 		template_name, context = symbol_routes_facade.html_summary()
 		return render_template(template_name, **context)
 
+	def _get_json_nodes(role, exact_match, request_args):
+		only_ssl = None
+		if 'only_ssl' in request_args:
+			only_ssl = True
+
+		order = request_args.get('order', None)
+
+		limit = int(request_args.get('limit', 0))
+
+		return jsonify(symbol_routes_facade.json_nodes(role=role, exact_match=exact_match, only_ssl=only_ssl, limit=limit, order=order))
+
```

Comment:
It seems ``role`` is always a required arg?

Reply:
Yes, you are right.
It filters the `/api/symbol/nodes/peer` and `/api/symbol/nodes/api` endpoint.